config-tool/pkg/lib/shared/structs.go

```diff
@@ -27,6 +27,8 @@ type DistributedStorageArgs struct {
 	AccessKey   string `default:"" validate:"" json:"access_key,omitempty" yaml:"access_key,omitempty"`
 	SecretKey   string `default:"" validate:"" json:"secret_key,omitempty" yaml:"secret_key,omitempty"`
 	BucketName  string `default:"" validate:"" json:"bucket_name,omitempty" yaml:"bucket_name,omitempty"`
+
+	Signature string `default:"s3v2" validate: "" json:"signature_version,omitempty" yaml:"signature_version,omitempty"`
```

Comment:
Do we need this since we're not doing any other validations related to the sig version?

Reply:
@ibazulic  without the definition the config-validator will reject if it's defined (had the same with `region_name`)

Reply:
Gotcha!